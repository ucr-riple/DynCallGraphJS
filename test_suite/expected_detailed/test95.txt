{
    'global (test95.js:1:1:17:5)': [
        'anon (test95.js:1:2:17:2)'
    ],
    'anon (test95.js:16:3:16:16)': [
        'anon (test95.js:3:10:6:6)'
    ],
    'anon (test95.js:4:7:4:10)': [
        'anon (test95.js:7:10:9:6)'
    ]
}