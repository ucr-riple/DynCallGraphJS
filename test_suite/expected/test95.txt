{ '(test95.js:1:1:17:5)':
   [ '(test95.js:1:2:17:2)' ],
  '(test95.js:16:3:16:16)':
   [ '(test95.js:3:10:6:6)' ],
  '(test95.js:4:7:4:10)':
   [ '(test95.js:7:10:9:6)' ] }